README.md

```diff
@@ -75,29 +81,42 @@ ReactDOM.render(
 )
 ```
 
-Now you can read from `state.routing.path` to get the URL. It's far more likely that you want to change the URL more often, however. You can use the `pushPath` action creator that we provide:
+Now you can read from `state.routing.location.pathname` to get the URL. It's far more likely that you want to change the URL more often, however. You can use the `push` action creator that we provide:
 
 ```js
-import { pushPath } from 'redux-simple-router'
+import { routeActions } from 'redux-simple-router'
 
 function MyComponent({ dispatch }) {
-  return <Button onClick={() => dispatch(pushPath('/foo'))}/>;
+  return <Button onClick={() => dispatch(routeActions.push('/foo'))}/>;
 }
 ```
 
-This will change the state, which will trigger a change in react-router. Additionally, if you want to respond to the path update action, just handle the `UPDATE_PATH` constant that we provide:
+This will change the state, which will trigger a change in react-router. Additionally, if you want to respond to the path update action, just handle the `UPDATE_LOCATION` constant that we provide:
 
 ```js
-import { UPDATE_PATH } from 'redux-simple-router'
+import { UPDATE_LOCATION } from 'redux-simple-router'
 
 function update(state, action) {
   switch(action.type) {
-  case UPDATE_PATH:
+  case UPDATE_LOCATION:
     // do something here
   }
 }
 ```
 
+### Server-side usage
+
+Using redux-simple-router on the server works nearly the same as using it on the client-side - except for the creation of the history. To make that work correctly, you have to hydrate the history with the current location passed from the server. This is necessary for redux-simple-router to determine the correct initial state.
+
+```js
+import { createHistory } from 'history'
+
+// Creates a memoryHistory with a necessary initial pathname.  
+const history = createHistory({ entries: [ req.url ] });
```

Comment:
Can you remove this section? I don't think this is the right way to do server-rendering. Why do you have a history object at all? You should be using the lower-level `match` API directly.